tests/test_yaml2sbml/ode_input_invalid_time_2.yaml

```diff
@@ -0,0 +1,9 @@
+time:
```

Comment:
what makes this model invalid?

Reply:
that time is specified as a list. basically the `-` in front of variable.

Reply:
wow! so if there is no dash before `variable` it is valid?

Reply:
yep :)

Reply:
We currently do not catch, if there is more than one variable assigned, but that would be an other issue :)